cnd/src/http_api/orderbook.rs

```diff
@@ -52,10 +55,11 @@ struct TakeHerc20HbitOrderBody {
 #[derive(Serialize)]
 struct Herc20HbitOrderResponse {
     #[serde(with = "asset::bitcoin::sats_as_string")]
-    buy_quantity: asset::Bitcoin,
-    sell_token_contract: ethereum::Address,
-    sell_quantity: asset::Erc20Quantity,
-    absolute_expiry: u32,
+    btc_quantity: asset::Bitcoin,
+    erc20_token_contract: ethereum::Address,
+    erc20_quantity: asset::Erc20Quantity,
+    alpha_expiry: u32,
+    beta_expiry: u32,
```

Comment:
My gut feeling is that using alpha/beta here instead of associating the expiries with the asset is going to bite us. Once we add position (buy/sell), devs are going to have to constantly map these expiry fields to bitcoin/ethereum depending on position. I'd prefer to see these named `bitcoin_absolute_expiry`  and `ethereum_absolute_expiry` to make the code easier to maintain and less fragile.